db/seeds.d/100-installation_media.rb

```diff
@@ -5,7 +5,8 @@
 # Installation media: default mirrors
 Medium.without_auditing do
   [
-    { :name => "CentOS mirror",        :os_family => "Redhat",  :path => "http://mirror.centos.org/centos/$major/os/$arch" },
+    { :name => "CentOS mirror (7.x)",  :os_family => "Redhat",  :path => "http://mirror.centos.org/centos/$major/os/$arch" },
+    { :name => "CentOS mirror (8.x)",  :os_family => "Redhat",  :path => "http://mirror.centos.org/centos/$major/BaseOS/$arch/kickstart" },
```

Comment:
How about:
```suggestion
    { :name => "CentOS 7 mirror",  :os_family => "Redhat",  :path => "http://mirror.centos.org/centos/$major/os/$arch" },
    { :name => "CentOS 8 mirror",  :os_family => "Redhat",  :path => "http://mirror.centos.org/centos/$major/BaseOS/$arch/kickstart" },
```
(and fix the migration accordingly)

Reply:
It's shorter, why not. Rebased.